docroot/modules/custom/mass_utility/mass_utility.module

```diff
@@ -131,7 +131,7 @@ function mass_utility_form_node_form_alter(&$form, FormStateInterface $form_stat
       '#type' => 'details',
       '#title' => t('SiteImprove Groups'),
       '#group' => 'advanced',
-      '#weight' => 90,
+      '#weight' => 90
```

Comment:
```suggestion
      '#weight' => 90,
```
This is a PHP coding standard recommendation

Reply:
Thank you.  That's corrected.